pkg/ottl/contexts/ottllog/README.md

```diff
@@ -11,6 +11,8 @@ The following fields are the exception.
 
 | path                                           | field accessed                                                                       | type                                                                    |
 |------------------------------------------------|--------------------------------------------------------------------------------------|-------------------------------------------------------------------------|
+| tmp                                            | the value of the current transform context's temporary storage                       | pcommon.Map                                                             |
```

Comment:
We should add this for the other contexts as well.

Reply:
I'll start adding it to the other context since it seems like we are in agreement.

Reply:
Done